src/com/smilingrob/plugin/robohexar/JavaDocParser.java

```diff
@@ -80,13 +80,12 @@ private boolean hasTestAnnotation(@NotNull PsiMethodImpl psiMethod) {
         }
         return false;
     }
-
     /**
-     * Determines whether or not the given method is a getter or a setter.
-     *
-     * @param psiMethodImpl method node.
-     * @return true if the given method is a getter or a setter.
-     */
+    * Determines whether or not the given method is a getter or a setter.
+    *
+    * @param psiMethodImpl method node.
+    * @return true if the given method is a getter or a setter.
+    */
```

Comment:
I removed the javadoc to set off RoboHexar for testing.  I put them back and they were off.  I used github directly.